notebooks/region_search/Region Searching Workbook.ipynb

```diff
@@ -1754,6 +1755,108 @@
     "df[\"ut\"]"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 129,
+   "id": "b45d3fa2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# As a reality check, let's make sure there aren't two identical dataId + UT instances\n",
+    "def look_for_duplicates(df):\n",
```

Comment:
Pandas series has a [is_unique()](https://pandas.pydata.org/docs/reference/api/pandas.Series.is_unique.html) method, so you could simplify this with
```
def has_duplicates(df):
   if not df["uri"].is_unique():
       assert AttributeError(...)
   if not df["data_id"].is_unique():
       assert AttributeError(...)
   # etc...
```